rasa_core/tracker_store.py

```diff
@@ -13,6 +13,8 @@
 import six.moves.cPickle as pickler
 from typing import Text, Optional, List
 
+from pymongo import ReturnDocument
```

Comment:
please move this import to the retrieve function, since it's only actually needed for the MongoTrackerStore